### DISCUSSION TITLE
PF13 - datatable slowness

### DISCUSSION THREAD

**Discussion by luccotta** [primefaces/community#2799] [PrimeFaces] [answered]
Hello, I just upgraded from PF8 to PF13, and I noticed that some pages are suddenly much slower, like more than 10 seconds every time the bean (`@ViewScoped`) gets called.

Apparently, the more columns I have in an `<p:columns` component inside a `<p:datatable`, the slower any request to the bean becomes. Any `p:commandButton` calling any method in the bean, even if the method is empty, the request takes a long time.

 Does that make any sense? Why would this happen? Any way to fix?
 
 Thanks

**EDIT**: I solved the issue by adding `process="@this"` to my` p:commandButtons`. So apparently the data table is being processed in PF13 on submit and in PF8 that did not happen? Should I add any attribute to the datatable to avoid this?

**Comment by melloware** [comment, marked as answer]
You can try this: https://primefaces.github.io/primefaces/vdldoc/p/dataTable.html#skipChildren
